app/presenters/licences/view-licence.presenter.js

```diff
@@ -132,6 +133,12 @@ function _endDate (expiredDate) {
   return formatLongDate(expiredDate)
 }
 
+function _getAuthRoles (auth) {
+  return auth?.credentials?.roles?.map((role) => {
+    return role?.role
+  }) || null
```

Comment:
Having experienced being thrown at a Node.js project when I knew very little about the language I try and keep things relatively simple.

In this case, I'd just roll it back one step and suggest this

```suggestion
  const roles = auth?.credentials?.roles?.map((role) => {
    return role?.role
  })

  return roles || null
```